examples/in-app-message-example/src/App.tsx

```diff
@@ -2,7 +2,7 @@ import { NetworkStatus } from "@knocklabs/client";
 import {
   Banner,
   CardView,
-  KnockInAppChannelProvider,
+  KnockInAppMessagesChannelProvider,
```

Comment:
I thought "InAppChannelProvider" works fine fwiw 😅 

Not blocking though, want to prioritize getting these into `next` soon!

Reply:
Yeah, just went for being completely consistent if overly verbose. Can adjust if we want during a final qa pass